pyqtorch/utils.py

```diff
@@ -5,11 +5,14 @@
 
 import torch
 
-from pyqtorch.matrices import DEFAULT_MATRIX_DTYPE
+from pyqtorch.matrices import DEFAULT_MATRIX_DTYPE, DEFAULT_REAL_DTYPE
 
 State = torch.Tensor
 Operator = torch.Tensor
 
+ATOL = 1e-07 if DEFAULT_REAL_DTYPE == torch.float32 else 1e-014
+GRADCHECK_ATOL = 1e-01 if DEFAULT_REAL_DTYPE == torch.float32 else 1e-014
```

Comment:
that's a huge difference?

Reply:
yeah but grad_check fails otherwise, we could not use gradcheck and do it manually like in other tests

Reply:
I guess let's keep it like this. 
As you pointed out in the DM it seems to be standard. I just didn't except e-14 -> e-1...

Reply:
i think this is definitely a failure on the torch side

Reply:
its manually tested using 1e-06 now